doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+workaround solution, and you can install Ubuntu shell on your Windows system
+after following the tutorial below:
+
+https://github.com/MicrosoftDocs/WSL/blob/live/WSL/install-win10.md
+
+In your command prompt, run ``ubuntu`` to transfer to Linux terminal, and
+follow the Debian/Ubuntu tutorial above to install the dependencies, and then
+you can run ``make html`` to build. (Note that you also have to install
+``make`` via ``apt-get install make``.)
+
+If you want to change the directory in your prompt to your working folder of
+SymPy in the Windows file system, you can prepend ``cd /mnt/`` to your file
+path in Windows, and run in your shell to navigate to the folder. (Also note
+that Linux uses ``/`` instead of ``\`` for file paths.)
+
+This method provides better compatibility than Cygwin or MSYS2 and more
+convenience than a virtual machine if you partially need a Linux environment
+for your workflow, however this method is only viable for Windows 10 64-bit
+users.
+
+2. Build the Documentation
+--------------------------
+
+The documentation can be built by running the ``makefile`` in the ``doc``
+subdirectory.
+
+To start, in your preferred web browser, use the drop down menu and select
+“open file” to navigate into the sympy/doc folder saved on your computer. In
+the doc folder, select the _build folder, then the html folder, and in the html
+folder, open the index.html file.
+
+To build the HTML documentation, run::
+
+  cd doc
+
+  make html
+
+This builds a local version of the documentation in ``doc/_build/html`` in your
+web browser.
+
+Open ``_build/html/index.html``.
+
+3. Make a Contribution
+----------------------
+
+For in-depth instructions on how to contribute to SymPy’s code base including
+coding conventions, creating your environment, picking an issue to fix, and
+opening a pull request, please read our full `Development Workflow
+<https://github.com/sympy/sympy/wiki/Development-workflow>`_ guide.
+
+Best Practices for Writing Documentation
+========================================
+
+Please follow these formatting, style, and tone preferences when writing
+documentation.
+
+Formatting Preferences
+----------------------
+
+**Math vs. Code**
+
+**Math:** text that is surrounded by dollar signs $ _ $ will be rendered as
+LaTeX math. Any text that is meant to appear as LaTeX math should be written as
+$math$. In the HTML version of the docs, MathJax will render the math.
+
+**Example**
+
+::
+
+  The Bessel $J$ function of order $\nu$ is defined to be the function
+  satisfying Bessel’s differential equation.
+
+**Code:** text that should be printed verbatim, such as code, should be
+surrounded by a set of double backticks ``like this``. (WORK IN PROGRESS: change
+single backticks to render as code via issue `#13519
+<https://github.com/sympy/sympy/issues/13519>`_.)
+
+**Example**
+
+::
+
+  To use this class, define the ``_rewrite`` and ``_expand`` methods.
```

Comment:
Would it be a good idea to recommend including parentheses when referring to methods and functions, so that is clear just be looking at it? e.g. `_rewrite()` instead.

Reply:
Yes, we should add that. There is a caveat that if you use `:func:` or `:meth:` they are added automatically (see https://github.com/sympy/sympy/pull/17698#issuecomment-538583439, which I believe Lauren will be adding to this document later).

Reply:
@asmeurer are you ready for me to add that cross referencing section to the guide in the next round of edits, or is it still a WIP?

Reply:
You can add it. I did make some edits to my comment in the past few days so make sure you have the latest version. If I think of anything else to add I will mention it here.